src/sentry/features/temporary.py

```diff
@@ -539,6 +539,8 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:widget-viewer-modal-minimap", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
     # Enabled unresolved issue webhook for organization
     manager.add("organizations:webhooks-unresolved", OrganizationFeature, FeatureHandlerStrategy.OPTIONS, api_expose=True)
+    # Enable metric alert issues (see: alerts create issues)
+    manager.add("organizations:workflow-engine-metric-alert-migration", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=False)
```

Comment:
chatted in slack, but for everyone else too -- should we have two (or more?) feature flags here? I was thinking we'd need 1 flag to enable dual writes, then another flag to enable different processing methods (shadow detector processing and shadow triggering actions)